sql/src/test/java/io/crate/integrationtests/ConditionalFunctionIntegrationTest.java

```diff
@@ -0,0 +1,29 @@
+package io.crate.integrationtests;
```

Comment:
I would remove this Integration tests since the case is already tested in the unit test, and integration tests add up to the time spent on CI builds.